package.json

```diff
@@ -73,10 +77,24 @@
     ]
   },
   "peerDependencies": {
+    "@edx/frontend-platform": "^1.5.2",
+    "@edx/paragon": "^12.0.5",
+    "prop-types": "^15.7.2",
     "react": "^16.13.1",
-    "@edx/frontend-platform": "^1.5.2"
+    "react-dom": "^17.0.1",
+    "react-redux": "^7.2.1",
+    "react-router-dom": "^5.2.0",
+    "react-loading-skeleton": "^2.1.1",
+    "redux": "^4.0.0"
   },
   "dependencies": {
-    "query-string": "^6.13.2"
+    "classnames": "^2.2.5",
+    "query-string": "^5.1.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.17",
```

Comment:
hahahaha we are putting them in only to remove them later with the rebrand. 😢 
(No issues, but I'll make a rebrand ticket for this repo now.)

Reply:
Thanks @raq929!